src/utils/index.ts

```diff
@@ -1008,3 +1008,10 @@ export function findEventRecord(
 export function padTicker(ticker: string): string {
   return padEnd(ticker, MAX_TICKER_LENGTH, '\u0000');
 }
+
+/**
+ * @hidden
+ */
+export function maximum(value1: BigNumber, value2: BigNumber): BigNumber {
```

Comment:
I don't think wrapping this call in a util adds any value. You should just use `BigNumber.max` directly